### ISSUE TITLE
installation failed

### ISSUE THREAD

**Issue by inuanouo** [Gourieff/comfyui-reactor-node#141]
### First, confirm

- [X] I have read the [instruction](https://github.com/Gourieff/comfyui-reactor-node/blob/main/README.md) carefully
- [X] I have searched the existing issues
- [X] I have updated the extension to the latest version

### What happened?

I can't install this model when using other models
![屏幕截图 2023-12-16 173600](https://github.com/Gourieff/comfyui-reactor-node/assets/139967597/df119ac2-5c7b-46e9-8aee-855e7b3c5c5c)

### Steps to reproduce the problem

Your workflow

### Sysinfo

ComfyUI win11 Microsoft Edge

### Relevant console log

```Shell
1
```

### Additional information

1

**Comment by Gourieff** [comment]
Try this https://github.com/Gourieff/comfyui-reactor-node/issues/136#issuecomment-1857890732

**Comment by inuanouo** [comment]
> 试试这个[#136（评论）](https://github.com/Gourieff/comfyui-reactor-node/issues/136#issuecomment-1857890732)

![屏幕截图 2023-12-16 185932](https://github.com/Gourieff/comfyui-reactor-node/assets/139967597/ef75e049-937a-4bfe-8566-a208fcd30298)
Why do I still fail?

**Comment by ckao10301** [comment]
> Try this [#136 (comment)](https://github.com/Gourieff/comfyui-reactor-node/issues/136#issuecomment-1857890732)

I'm having the same issue. Tried this, still failed.

**Comment by Gourieff** [comment]
Please update ReActor and run install.bat

**Comment by ckao10301** [comment]
I deleted, reinstalled, and ran install.bat. Still getting "install failed". Found this in the log, not sure if related: Cannot import C:\Users\ckao1\OneDrive\Desktop\stable swarm\StableSwarmUI\dlbackend\comfy\ComfyUI\custom_nodes\comfyui-reactor-node module for custom nodes: No module named 'torchvision.transforms.functional_tensor'

**Comment by lustfeind** [comment]
Problem still existing on Apple silicon. 

ModuleNotFoundError: No module named 'torchvision.transforms.functional_tensor'

**Comment by ckao10301** [comment]
Im on intel and nvidia

**Comment by ckao10301** [comment]
Update: I solved the torchvision module issue by running these commands in the comfy root. It was working fine before updating comfyui and custom nodes but looks like all these dependencies and versions cause things to break easily and quite a headache to solve.

python_embeded\python.exe -m pip uninstall torch torchvision torchaudio
python_embeded\python.exe -m pip install torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cu121

**Comment by wuzhuzhu** [comment]
mac m2 mps 
ModuleNotFoundError: No module named 'torchvision.transforms.functional_tensor'

**Comment by wuzhuzhu** [comment]
> mac m2 mps ModuleNotFoundError: No module named 'torchvision.transforms.functional_tensor'

it was working perfect before update

**Comment by lustfeind** [comment]
> #136 (comment)

Yes, it is still broken. Sadly.

**Comment by ckao10301** [comment]
Solution has been found for mac. check this: https://github.com/comfyanonymous/ComfyUI/issues/2316

**Comment by Gourieff** [comment]
> > 试试这个[#136（评论）](https://github.com/Gourieff/comfyui-reactor-node/issues/136#issuecomment-1857890732)
> 
> ![屏幕截图 2023-12-16 185932](https://private-user-images.githubusercontent.com/139967597/290988089-ef75e049-937a-4bfe-8566-a208fcd30298.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMyNzEyNzAsIm5iZiI6MTcwMzI3MDk3MCwicGF0aCI6Ii8xMzk5Njc1OTcvMjkwOTg4MDg5LWVmNzVlMDQ5LTkzN2EtNGJmZS04NTY2LWEyMDhmY2QzMDI5OC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIyMlQxODQ5MzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ZjUwNjRiMzg3YzBkYjZkMGM2M2MyN2EzMjIyODMyMDE1NTZhZjA1YWRiOTU1YjQ3MzQ0YWQ1M2E4OTk5YjMxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Tu_DGw2D2CWFxCkOBQ6d2kCWFwskLJE1QTtruBjo-F0) Why do I still fail?

Wrong folder to execute the command, you should have done one step up to the parent folder

Please update guys who has issues with Basicsr
It is built-in now (inside the ReActor's Node folder)

**Comment by gabrie** [comment]
![Snipaste_2023-12-24_15-31-12](https://github.com/Gourieff/comfyui-reactor-node/assets/539991/c501502c-152b-438b-8f04-815260e6eed2)
Thanks for the help, but I tried to uninstall and install the latest version of Reactor, from Comfy UI manager and from git clone, still encounter the "Import Failed" issue.

**Comment by rendomnet** [comment]
NOT WORKING! even after I copied insightface from main python folder and bunch of other libra to python_embeded.
now it asks for sklearn to be beuilded.

**Comment by alexandre-ruffer** [comment]
While installing the install.bat

Status: The 'insightface==0.7.3' distribution was not found and is required by the application
Collecting insightface==0.7.3
  Using cached insightface-0.7.3.tar.gz (439 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [18 lines of output]
      Traceback (most recent call last):
        File "C:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 353, in <module>
          main()
        File "C:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 118, in get_requires_for_build_wheel
          return hook(config_settings)
                 ^^^^^^^^^^^^^^^^^^^^^
        File "C:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\setuptools\build_meta.py", line 325, in get_requires_for_build_wheel
          return self._get_build_requires(config_settings, requirements=['wheel'])
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\setuptools\build_meta.py", line 295, in _get_build_requires
          self.run_setup()
        File "C:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\setuptools\build_meta.py", line 311, in run_setup
          exec(code, locals())
        File "<string>", line 11, in <module>
      ModuleNotFoundError: No module named 'Cython'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

**Comment by dronifydmv** [comment]
I went through the batch install and everything installed fine. However, I now have multiple nodes that are not installing instead of just the one that didn't work initially. 

![image](https://github.com/Gourieff/comfyui-reactor-node/assets/130697016/e1bd0801-8590-4cd8-b85c-382fffdef0d6)


Any help is much appreciated :-).

**Comment by enggre** [comment]
@dronifydmv I have had the same problem but I think I found a solution. (I worked for me at least)

Check this one out: 
https://github.com/Gourieff/comfyui-reactor-node?tab=readme-ov-file#iv-controlnet-aux-node-import-failed-error-when-using-with-reactor-node

Quote:
1: Close ComfyUI if it runs
2: Go to the ComfyUI root folder, open CMD there and run:
python_embeded\python.exe -m pip uninstall -y opencv-python opencv-contrib-python opencv-python-headless
python_embeded\python.exe -m pip install opencv-python==4.7.0.72
3: That's it!

**Comment by dronifydmv** [comment]
Thanks for sharing! I ended up getting it to work with a combination of things. I ended up deleting and reinstalling python and git. What a two day journey it was!  Sent from my iPhoneOn Jan 13, 2024, at 1:31 PM, enggre ***@***.***> wrote:﻿
@dronifydmv I have had the same problem but I think I found a solution. (I worked for me at least)
Check this one out:
https://github.com/Gourieff/comfyui-reactor-node?tab=readme-ov-file#iv-controlnet-aux-node-import-failed-error-when-using-with-reactor-node
Quote:
1: Close ComfyUI if it runs
2: Go to the ComfyUI root folder, open CMD there and run:
python_embeded\python.exe -m pip uninstall -y opencv-python opencv-contrib-python opencv-python-headless
python_embeded\python.exe -m pip install opencv-python==4.7.0.72
3: That's it!
